src/lib/queue.js

```diff
@@ -20,6 +21,9 @@ export default function(opts) {
             events.emit('job:run:init', {job});
 
             _id = job._id;
+
+            await apiClient.setState(_id, {state: 'RUNNING', error: undefined});
```

Comment:
`state` is currently [read-only from an API point of view](https://github.com/dicefm/timeturner/blob/master/src/schemas/Request.js#L26-L35)